webapp/src/components/ImageEditor/ImageCanvas.tsx

```diff
@@ -712,52 +712,23 @@ export class ImageCanvasImpl extends React.Component<ImageCanvasProps, {}> imple
     }
 
     protected drawBitmap(bitmap: pxt.sprite.Bitmap, x0 = 0, y0 = 0, transparent = false, alpha = !this.props.lightMode, target = this.canvas) {
-        if (!this.colors) {
```

Comment:
Was this lazy setting just not needed / always should have been using the props colors instead of this.colors?

Reply:
this was an array of color bytes that was derived from the color strings in props